app/views/hq/users/show.html.erb

```diff
@@ -18,7 +19,8 @@
   <%= pluralize(@user.inactive_sponsors.count.to_s, 'Inactive Sponsor') %>
   <% if @user.sponsors.count > 0 %>
     <div class="inactive_sponsors_index">
-      <%= render partial: 'hq/sponsors/sponsors.html.haml', locals: {sponsors: @user.inactive_sponsors} %>
+      <%= render partial: 'hq/sponsors/sponsors.html.haml',
+            locals: {sponsors: @user.inactive_sponsors.paginate(:page => params[:page])} %>
```

Comment:
User#show also uses the `_sponsors` partial here. 
To overwrite the default number of sponsors per page we ca use:
`@user.inactive_sponsors.paginate(:page => params[:page], :per_page => 15)}`